### PR TITLE
Fix load bug

### LINKED ISSUE #1022: Post-verify Home page issues
- Homepage stuck in endless loading when the user navigates away from the page and returns

- "Your data is complete" header should be removed 
Screen Shot 2021-01-12 at 11 28 35 AM

### PR DESCRIPTION
**Pull request by bamader**
Fixes infinite load state  for post-submit users  and prevents flashing for pre-submit users. Tested by login of non-submitted user, refreshing home, going to every tab nav page and  back, refreshing on  tab nav pages then going to home, then doing it all again after uploading and submitting. Didn't see anything weird on local. Closes #1022 .

### CODE REVIEW

**Review by jlhogan — APPROVED** [review]

File: client/src/containers/Home/PostSubmit.tsx [hunk 1]
```diff
@@ -126,7 +124,6 @@ export const PostSubmitHome: React.FC = () => {
 
   return (
     <div className="grid-container margin-top-4">
-      <h1 ref={h1Ref}>Your data is complete!</h1>
```
Comment (JordanGuinn) [inline client/src/containers/Home/PostSubmit.tsx:129]: Any particular reason this is getting pulled out?

**Review by JordanGuinn — APPROVED** [review]
LGTM 👍🏽👍🏽

File: client/src/containers/Home/PostSubmit.tsx (hunk 1 above)
Comment (bamader) [reply client/src/containers/Home/PostSubmit.tsx:129]: Ryan asked in the ticket for it to be taken out

Comment (JordanGuinn) [reply client/src/containers/Home/PostSubmit.tsx:129]: Heh, my bad - supposed I could've actually looked at the ticket 😅🤦🏽‍♂️

**Pull request merged**